README.md

```diff
@@ -1 +1,61 @@
-# Microverse_Week_rpt
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio: mobile version
+
+> This was Third Project and i have added a portfolio section showing the projects worked on. 
```

Comment:
Since the project does not have projects worked on yet: It is better not to add them in the readme file description. This will be more clear and easier to understand even in the future.